kernel/boot/Handoff.cpp

```diff
@@ -217,7 +173,7 @@ void Handoff::parseMultiboot2(Handoff* that, void* handoff)
             {
                 auto cmdline = (struct multiboot_tag_string *) tag;
                 rs232_printf("Multiboot2 cmdline: '%s'\n", cmdline->string);
-                that->_cmdline = (const char *)(cmdline->string);
+                that->_cmdline = (char *)(cmdline->string);
```

Comment:
Shouldn't need a cast if you change the type of `_cmdline` back to `const char *` (see my other comment)

Reply:
See other comment lol